### PR TITLE
Bug 2011939: Add a temporary hack to the azure 4.9-to-4.10 upgrade job

### PR DESCRIPTION
**Pull request by danwinship**
This may fix sporadic networking problems observed in e2e-azure-upgrade runs. See https://bugzilla.redhat.com/show_bug.cgi?id=2011939#c1 and https://github.com/openshift/machine-config-operator/pull/2799.

Assuming that this fixes the problem in the 4.9-to-4.10 upgrade jobs, we will need to come up with a more complicated fix for the actual OCP release, because applying the MachineConfigs here to _existing_ clusters would break them.

/hold
for testing

### CODE REVIEW

**Comment by dgoodwin** [comment]
If anyone else is curious the base64 text is:

```
#!/bin/sh

IFACE="$1"
STATUS="$2"
if [ "${IFACE}" = "eth0" -a "${STATUS}" = "up" ]; then
    echo "Overriding default MTU on ${IFACE}"
    ip link set "${IFACE}" mtu 1400
fi
```

Thanks a bunch Dan, fingers crossed.

/lgtm

Drop the hold when you're happy with the rehearse.

**Comment by danwinship** [comment]
so... the rehearsals are going pretty terribly, but I don't think that's my patch; it shouldn't even affect any of them except the 4.9-to-4.10 upgrade (and the logs seem to show that that's correct). Is this level of brokenness expected for these jobs?

**Comment by deads2k** [comment]
this rehearse looks interesting to watch: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/23224/rehearse-23224-periodic-ci-openshift-release-master-ci-4.10-upgrade-from-stable-4.9-e2e-azure-upgrade/1456600709888741376

File: ci-operator/config/openshift/release/openshift-release-master__ci-4.10-upgrade-from-stable-4.9.yaml [hunk 1]
```diff
@@ -43,6 +43,7 @@ tests:
     env:
       BASE_DOMAIN: ci2.azure.devcluster.openshift.com
       TEST_TYPE: upgrade-conformance
+      TEST_MTU_HACK: 1
```
Comment (stbenjam) [inline ci-operator/config/openshift/release/openshift-release-master__ci-4.10-upgrade-from-stable-4.9.yaml]: This should go before TEST_TYPE

**Comment by dgoodwin** [comment]
@danwinship could we get a little update for the status blurb on today's infra arch call? How are things looking with this.

**Comment by danwinship** [comment]
I don't really know... the job that I modifed isn't passing, but none of the others are either, so it seems unrelated to this PR?

**Comment by dgoodwin** [comment]
@danwinship I've just closed the bug, this issue appears to have resolved itself when I went back to check where we were at, the test is now at 98%+ pass rate. We don't have a reproducer to even chase. Suspect your theory may have been correct and fixed Azure side. In any case, I'd suggest closing if you're ok with it. Thank you for all the help chasing this.

**Comment by danwinship** [comment]
yay mysterious bugs that go away by themselves! or something...

**Pull request closed without merging**